.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,60 @@
+name: Prepare upcoming release
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+
+env:
+  NODE_VERSION: 22.x
+
+jobs:
+  prepare-release:
+    name: Release preparation steps
+    runs-on: ubuntu-latest
+    if: startsWith(github.event.pull_request.head.ref, 'release/')
+    steps:
+      - name: Extract version name
+        id: extract_version
+        run: |
+          BRANCH_NAME="${{ github.event.pull_request.head.ref }}"
+          VERSION="${BRANCH_NAME#release/}"
+          echo "Versión detectada: $VERSION"
```

Comment:
Let's translate this. Same below

Reply:
Oops, I've removed that line in 8d33b25. Same for the other workflow.